src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -28,9 +28,15 @@ import { TerminalContextKeys } from 'vs/workbench/contrib/terminal/common/termin
 
 const $ = dom.$;
 
-const FIND_FOCUS_CLASS = 'find-focused';
-const STATUS_ICON_WIDTH = 30;
-const SPLIT_ANNOTATION_WIDTH = 30;
+const enum Class {
```

Comment:
Seeing `Class` in code looks wrong as it's a keyword when lowercase. How about `CssClass` instead?